src/Popover/Popover.js

```diff
@@ -38,37 +38,37 @@ const Popover = React.createClass({
     animation: React.PropTypes.func,
 
     /**
-     * If true, the popover will hide when the anchor scrolls off the screen
+     * If true, the popover will hide when the anchor scrolls off the screen.
      */
     autoCloseWhenOffScreen: React.PropTypes.bool,
 
     /**
-     * If true, the popover (potentially) ignores targetOrigin
-     * and anchorOrigin to make itself fit on screen,
+     * If true, the popover (potentially) ignores `targetOrigin`
+     * and `anchorOrigin` to make itself fit on screen,
      * which is useful for mobile devices.
      */
     canAutoPosition: React.PropTypes.bool,
 
     /**
-     * Use this property to render your component inside the `Popover`.
+     * Use this property to render your component inside the `Popover` element.
```

Comment:
:+1: I'm good with either. The first is perhaps more technically accurate, but the second is more succinct, while still describing what the purpose of the property is (in either case, assuming users know that it is normally populated through composition).